{
  "contest": 420,
  "handle": "BugsFinder0x",
  "risk": "3",
  "title": "Impossible to call allowDaoFunds() function due to a concept error",
  "issueId": 699,
  "issueUrl": "https://github.com/code-423n4/2024-07-traitforge-validation/issues/699"
}